app/services/bcrypt.js

```diff
@@ -0,0 +1,10 @@
+const bcrypt = require('bcrypt');
+const errors = require('../errors');
+
+exports.crypt = hash =>
+  bcrypt
+    .genSalt(10)
```

Comment:
The salt rounds should be passed by parameters or included in some constant as default value